src/main/resources/static/js/main.js

```diff
@@ -575,4 +575,60 @@ function generatePWD()
 	password = shuffleString(password);
 
 	return password;
-}
\ No newline at end of file
+}
+
+$( document ).ready(function() {
+
+	// let's Hide Items
+	$( ".cc-contactpop form").hide();
+	$( ".cc-contactpop i").hide();
+	$( ".cc-contactpop p").hide();
+
+	// Show on elements ".slideDown"
+	$( ".cc-contactpop" ).click(function() {
+		$( ".cc-contactpop form").slideDown(500)
+		$( ".cc-contactpop p").slideDown(500)
+		$( ".cc-contactpop i").slideDown(500)
+	});
+
+	// Close Hidden Part
+	$( ".cc-contactpop i" ).click(function() {
+		$(this).slideUp(500)
+		$( ".cc-contactpop p").slideUp(500)
+		$( ".cc-contactpop form").slideUp(500)
+		return false
+	});
+
+});
+
+// Add show/hide button
+var sShowHideBtn = '<button class="contact-button-link show-hide-contact-bar"><span class="fa fa-angle-left"></span></button>';
+oContainer.append(sShowHideBtn);
+
+var i;
+for ( i in settings.buttons ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Restrict what this loop acts on by testing each property. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AForIn)